### ISSUE TITLE
Nov. 22nd Workshop import (and subsequent paste) borked

### ISSUE THREAD

**Issue by icy-** [awgil/ffxiv_visland#86]
![image](https://github.com/user-attachments/assets/2e490aa0-eed8-4a1e-a259-216c60ca0955)
Hello, I have a bug here in latest version.  Here is today's OC source, with resulting select all and copy looking like 

```
Cycle 5
First 3 Workshops
:OC_Isloaf: Isloaf (4h)
:OC_Peperoncino: Peperoncino (6h)
:OC_Isloaf: Isloaf (4h)
:OC_Peperoncino: Peperoncino (6h)
:OC_Isloaf: Isloaf (4h)
.
4th Workshop
:OC_Brush: Brush (4h)
:OC_SpruceRoundShield: Spruce Round Shield (8h)
:OC_Brush: Brush (4h)
:OC_SpruceRoundShield: Spruce Round Shield (8h)
Grooveless Value
6267
With 37 Groove
8283 :OC_SeafarerCowrie:
Cycle 6
First 3 Workshops
:OC_BoiledEgg: Boiled Egg (4h)
:OC_Bouillabaisse: Bouillabaisse (8h)
:OC_BoiledEgg: Boiled Egg (4h)
:OC_Bouillabaisse: Bouillabaisse (8h)
.
4th Workshop
:OC_CulinaryKnife: Culinary Knife (4h)
:OC_ScaleFingers: Scale Fingers (8h)
:OC_BoiledEgg: Boiled Egg (4h)
:OC_Bouillabaisse: Bouillabaisse (8h)
Grooveless Value
7486
With 45 Groove
10098 :OC_SeafarerCowrie:
```

Clicking the OC import button resulted in this
![image](https://github.com/user-attachments/assets/1ef0e46e-09bf-4a1a-9f60-174c61040bc9)

We see that Cycle 5's workshops were split incorrectly and the icons are wrong.  Cycle 6's icons are wrong too.  Applying to the actual cycle5+cycle6 we see that actually the internal items are wrong:

![image](https://github.com/user-attachments/assets/d9bca661-2381-4c4a-84ef-620341bd49a4)
![image](https://github.com/user-attachments/assets/4cb52c34-8f1a-4925-893c-a87cd4988527)

So some kind of serious offset bug ^_^

In log for version info it says > [11/22/2024 9:43:55 AM -05:00] This is ECommons v0.0.0.125 (release build without forms) and visland v0.0.0.125. Hello!

thx!

**Comment by icy-** [comment]
Just the first three lines are enough to reproduce the bug
```
Cycle 5
First 3 Workshops
:OC_Isloaf: Isloaf (4h)
```

**Comment by Jaksuhn** [comment]
fixed in [dd7f229](https://github.com/awgil/ffxiv_visland/commit/dd7f229856f050ea0fbeaffc09888b7aedd56031)
